src/server/routes/callback.js

```diff
@@ -110,7 +110,7 @@ export default async (req, res, options, done) => {
           // e.g. option to send users to a new account landing page on initial login
           // Note that the callback URL is preserved, so the journey can still be resumed
           if (isNewUser && pages.newUser) {
-            return redirect(pages.newUser)
+            return redirect(`${pages.newUser}?callbackUrl=${encodeURIComponent(callbackUrl)}`)
```

Comment:
Suggestion so we check if the URL already has a ? in it (and adds the option with & instead).

```suggestion
            return redirect(`${pages.newUser}${pages.newUser.includes('?') ? '&' : '?'}callbackUrl=${encodeURIComponent(callbackUrl)}`)
```